### PR TITLE
[ZT] Fleet status

### PR DESCRIPTION
**Pull request by ranbel**
PCX-5851

### CODE REVIEW

File: content/cloudflare-one/analytics/dex.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+---
+
+# Digital Experience Monitoring
+
+Digital Experience Monitoring provides visibility into device, network, and application performance across your Zero Trust organization.  This information enables you to understand the state of your WARP client deployment and quickly resolve issues impacting end-user productivity.
+
+To view an overview of all enrolled devices, go to **DEX** > **Analytics**. You will see the following metrics:
+
+- **Devices connected by colo**: Number of devices that are currently connected to a given [Cloudflare data center](https://www.cloudflarestatus.com/).
+- **Connectivity status**: Percentage of devices that are currently connected, [disconnected](/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-settings/#lock-warp-switch), [paused](/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-settings/#admin-override), or connecting.
```
Comment (ranbel) [inline content/cloudflare-one/analytics/dex.md]: Can we clarify what these statuses mean?

File: content/cloudflare-one/analytics/dex.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+To view an overview of all enrolled devices, go to **DEX** > **Analytics**. You will see the following metrics:
+
+- **Devices connected by colo**: Number of devices that are connected to a given [Cloudflare data center](https://www.cloudflarestatus.com/).
+- **Connectivity status**: Percentage of devices in a given WARP client state.
+{{<table-wrap>}}
+| Status            | Description |
+|-------------------|-------------|
+| Connected    |  WARP has successfully established a connection to the Cloudflare global network.  |
+| Disconnected | WARP has been intentionally or unintentionally disconnected from the Cloudflare global network.  |
+| Paused       | A user or administrator has taken an explicit action to temporarily disconnect WARP, for example by entering an [Admin Override](/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-settings/#admin-override) code.|
```
Comment (ranbel) [inline content/cloudflare-one/analytics/dex.md]: need to clarify what actions trigger the Paused state

Comment (ranbel) [reply content/cloudflare-one/analytics/dex.md]: From Shiv: Client can go in to a paused switch when Switch Locked = False and Auto Connect > 0

Comment (ranbel) [inline content/cloudflare-one/analytics/dex.md]:
```suggestion
| Paused       | A user or administrator has taken an explicit action to temporarily turn off WARP, for example by entering an [Admin Override](/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-settings/#admin-override) code.|
```

Comment (ranbel) [inline content/cloudflare-one/analytics/dex.md]:
```suggestion
| Paused       | A user or administrator has taken an explicit action to temporarily turn off WARP, for example by entering an [Admin Override](/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-settings/#admin-override) code. Paused clients will [auto-connect](/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-settings/#auto-connect) after a timeout period. |
```

**Review by patriciasantaana — APPROVED** [review]

**Pull request merged**